tensorboard/components/tf_ng_tensorboard/header/containers/header.component.ts

```diff
@@ -13,29 +13,40 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 import {Component} from '@angular/core';
+import {MatTabChangeEvent} from '@angular/material/tabs';
+import {MatSelectChange} from '@angular/material/select';
 import {Store, select, createSelector} from '@ngrx/store';
 import {combineLatest, of} from 'rxjs';
 
 import {State, getActivePlugin, getPlugins} from '../../core/core.reducers';
 import {changePlugin} from '../../core/core.actions';
 
+import {PluginMetadata, PluginId} from '../../types/api';
+
 /** @typehack */ import * as _typeHackRxjs from 'rxjs';
 
+interface UiPluginMetadata extends PluginMetadata {
+  id: PluginId;
+}
+
 const selectPlugins = createSelector(
   getPlugins,
```

Comment:
q: I see that `getPlugins` and `selectPlugins` are different things but
are both “selectors”. Is there some naming convention here, such that if
I were more familiar I could easily say “oh, `getFoo` does X while
`selectFoo` does Y” without having to know the details?

Reply:
I mis-remembered the code somewhere and took this as a convention. I changed everything to read `get` instead. Thanks for pointing this out!